1.1.0/jessie/build-projectjson/README.md

```diff
@@ -8,24 +8,43 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
 [![Downloads from Docker Hub](https://img.shields.io/docker/pulls/microsoft/aspnetcore-build.svg)](https://hub.docker.com/r/microsoft/aspnetcore-build)
 [![Stars on Docker Hub](https://img.shields.io/docker/stars/microsoft/aspnetcore-build.svg)](https://hub.docker.com/r/microsoft/aspnetcore-build)
 
-This image contains:
-
-- [.NET Command Line Interface (CLI)](https://github.com/dotnet/cli) so that you can create, build and run your .NET Core applications.
-- A Nuget package cache for the ASP.NET Core libraries.  This will significantly improve the initial package restore
-  performance when building ASP.NET Core application.
-- [Node.js](https://nodejs.org)
-- [Bower](https://bower.io/)
-- [Gulp](http://gulpjs.com/)
-
 ## Supported tags
 
 - [`1.1.0-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-projectjson/Dockerfile)
 - [`1.1.0-msbuild` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-msbuild/Dockerfile)
 - [`1.0.1-msbuild`, (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build-msbuild/Dockerfile)
 - [`1.0.1`, (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build-projectjson/Dockerfile)
 
+## What is ASP.NET Core?
+
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud based internet connected applications, such as web apps, IoT apps and mobile backends. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at [GitHub](https://github.com/aspnet). 
+
+This image contains:
+
+- [.NET Core SDK](https://github.com/dotnet/cli) so that you can create, build and run your .NET Core applications.
+- A Nuget package cache for the ASP.NET Core libraries.  This will significantly improve the initial package restore performance when building ASP.NET Core application.
+- [Node.js](https://nodejs.org)
+- [Bower](https://bower.io/)
+- [Gulp](http://gulpjs.com/)
+
 ## Example Usage
 
+### Build on run
+
+You can use this container to compile your application when it runs. If you use the [Visual Studio tooling](https://blogs.msdn.microsoft.com/webdev/2016/11/16/new-docker-tools-for-visual-studio/) to setup CI/CD to Azure Container Service then this method of using the build container is used.
+
+1. Run the build container, mounting your code and output directory, and publish your app:
```

Comment:
Seems odd to have a numbered list of one item. Maybe leave as a normal paragraph?